telegram/chat.py

```diff
@@ -96,6 +100,7 @@ def __init__(self,
                  invite_link=None,
                  pinned_message=None,
                  permissions=None,
+                 slow_mode_delay=None,
```

Comment:
please add at then end of the constructor. It's annoying, I know, but people still use args...